### PR TITLE
Update menus

### PR DESCRIPTION
**Pull request by szaimen**

### CODE REVIEW

**Comment by szaimen** [comment]
This is ready for review since I only moved thinks around.

File: menu/additional_apps.sh [hunk 1]
```diff
@@ -32,11 +33,16 @@ choice=$(whiptail --title "Which apps do you want to install?" --checklist "Auto
...
 
 case "$choice" in
-    *"Fail2ban"*)
+    *"Fail2ban "*)
         clear
         print_text_in_color "$ICyan" "Downloading Fail2ban.sh..."
         run_script APP fail2ban
     ;;&
+    *"Fail2ban-Statuscheck"*)
+        clear
```
Comment (enoch85) [inline menu/additional_apps.sh:42]: Please add a check if Fail2ban is installed.

Thanks!

Comment (szaimen) [reply menu/additional_apps.sh:42]: done 👍

File: menu/additional_apps.sh [hunk 2]
```diff
@@ -33,11 +34,21 @@ choice=$(whiptail --title "Which apps do you want to install?" --checklist "Auto
...
+        if is_this_installed fail2ban
+        then
+            fail2ban-client status nextcloud && fail2ban-client status sshd
+            iptables -L -n
+        else
+            msg_box "Fail2ban isn't installed. Please run 'sudo bash /vasr/scripts/menu.sh' to install it."
```
Comment (szaimen) [inline menu/additional_apps.sh:49]: @enoch85 I guess this is a typo?

**Pull request merged**